hw_34_final_project/src/js/helpers.js

```diff
@@ -0,0 +1,94 @@
+export const Helpers = {
+
+    async getWeatherbyCoords({ lat, lon }) {
+        let data;
+
+        try {
+            let response = await fetch(`https://api.openweathermap.org/data/2.5/weather?lat=${lat}&lon=${lon}&appid=817ee402db4510ccfef28a1daa72266d&units=metric`);
```

Comment:
need to save base URL in kinda environment configs. That way we get rid of repetition and bring easily way to change the domain name